### PR TITLE
chore: Updates fault docs for 3.0.0

### PR DESCRIPTION
**Pull request by neelanjan00**
## Proposed changes

Updates fault docs for 3.0.0

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: mkdocs/docs/experiments/categories/aws/ec2-stop-by-id.md [hunk 1]
```diff
@@ -54,24 +54,24 @@ When the MANAGED_NODEGROUP is enable then the experiment will not try to start t
 
     ??? note "View the Minimal RBAC permissions"
 
-        [embedmd]:# (https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/kube-aws/ec2-terminate-by-id/rbac.yaml yaml)
+        [embedmd]:# (https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/kube-aws/ec2-stop-by-id/rbac.yaml yaml)
```
Comment (ispeakc0de) [inline mkdocs/docs/experiments/categories/aws/ec2-stop-by-id.md:57]: we don't have rbac at this url. Please check at other places as well

File: mkdocs/docs/experiments/categories/aws/ec2-stop-by-tag.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
-- It causes termination of an EC2 instance by tag before bringing it back to running state after the specified chaos duration.
+- It causes stopping of an EC2 instance by tag before bringing it back to running state after the specified chaos duration.
 - It helps to check the performance of the application/process running on the ec2 instance.
 When the MANAGED_NODEGROUP is enable then the experiment will not try to start the instance post chaos instead it will check of the addition of the new node instance to the cluster.
 
-!!! tip "Scenario: Terminate EC2 Instance"    
-    ![EC2 Terminate By Tag](../../images/ec2-terminate.png)
+!!! tip "Scenario: Stop EC2 Instance"    
+    ![EC2 Stop By Tag](../../images/ec2-stop.png)
```
Comment (ispeakc0de) [inline mkdocs/docs/experiments/categories/aws/ec2-stop-by-tag.md:8]: Update name here as well

Comment (neelanjan00) [reply mkdocs/docs/experiments/categories/aws/ec2-stop-by-tag.md:8]: It's already updated, right? 🤔

File: mkdocs/docs/experiments/categories/aws/ec2-stop-by-id.md (hunk 1 above)
Comment (neelanjan00) [reply mkdocs/docs/experiments/categories/aws/ec2-stop-by-id.md]: Removed all embedmd links for missing RBACs

**Comment by Saranya-jena** [comment]
@ispeakc0de @neelanjan00 Can you update this PR?

**Review by ispeakc0de — APPROVED** [review]

**Pull request merged**